app/controllers/api/v1/subject_sets_controller.rb

```diff
@@ -86,7 +86,6 @@ def destroy_relation(resource, relation, value)
   private
 
   def remove_linked_set_member_subjects(resource, set_member_subjects)
-    SubjectWorkflowCount.where(set_member_subject: set_member_subjects).delete_all
```

Comment:
What's the reason for not liking a pg cascade? Too magical or is there a performance reason I'm unaware of? I don't think there's any need to keep a counter for a subject/workflow that no longer exists. We should generally either do soft-deletions or just remove everything associated (imho).